add blogpost on migrating away from the Jaeger exporter

codeboten opened the pull request:
This is to help users that upgrade to v0.85.0 of the OTel Collector.

svrnm commented on content/en/blog/2023/jaeger-exporter-collector-migration.md:
```diff
@@ -0,0 +1,73 @@
+linkTitle: Migrating away from the Jaeger exporter in the Collector
+date: 2023-09-14
+author: >-
+  [Alex Boten](https://twitter.com/codeboten) (Lightstep)
```
I think so far we always linked github accounts.

codeboten replied:
I just followed the example i copied here: https://github.com/open-telemetry/opentelemetry.io/blob/1fb0f484fbac6df101139f7db789fbb306eb0f6e/content/en/blog/2023/outreachy-may-cohort.md?plain=1#L6

I don't have a problem changing it though.

svrnm replied:
Ah... I missed that one. Whatever you prefer, I personally prefer the github accounts because (a) they give people direct insights into your work with OpenTelemetry and (b) your twitter link only works for people authenticated with the platform

codeboten replied:
np. I like the idea of being consistent :)

svrnm commented on content/en/blog/2023/jaeger-exporter-collector-migration.md:
```diff
@@ -0,0 +1,73 @@
+---
+title: Migrating away from the Jaeger exporter in the Collector
+linkTitle: Migrating away from the Jaeger exporter in the Collector
+date: 2023-09-14
```
Let's aim for some time next week, we still have another blog post in the loop for Monday next week, maybe Wednesday/Thursday?

codeboten replied:
That's fine, the release was done Monday and there was an issue raised yesterday about it, which is why I figured the sooner the better :)

svrnm commented on content/en/blog/2023/jaeger-exporter-collector-migration.md:
````diff
@@ -0,0 +1,73 @@
+    exporters: [otlp/jaeger]
+```
+
+Depending on your version of Jaeger, you may need to set the following
+environment variable `COLLECTOR_OTLP_ENABLED=true`.
````
can you put a bound here like "since version X this is not required anymore"

codeboten replied:
This is a good question, maybe @pavolloffay or @yurishkuro can confirm what the right versions are here. I onlny tested this w/ the all-in-one container versions 1.35 & 1.41.

yurishkuro replied:
from v1.46.0 there is no need to set COLLECTOR_OTLP_ENABLED=true

svrnm commented on content/en/blog/2023/jaeger-exporter-collector-migration.md:
```diff
@@ -0,0 +1,73 @@
+repository.
+
+If either of these options do not work for your use-case, please reach out to
+the #otel-collector community via [CNCF Slack](https://slack.cncf.io) or open an
```
```suggestion
the [#otel-collector](https://cloud-native.slack.com/archives/C01N6P7KR6W) community via [CNCF Slack](https://slack.cncf.io) or open an
```

mx-psi commented on content/en/blog/2023/jaeger-exporter-collector-migration.md:
````diff
@@ -0,0 +1,73 @@
+
+```yaml
+exporters:
+  otlp/jaeger: # Jaeger supports OTLP directly
````
Maybe worth explaining the port (and mentioning in passing this is OTLP/gRPC specifically)
```suggestion
  otlp/jaeger: # Jaeger supports OTLP directly. The default port for OTLP/gRPC is 4317
```

mx-psi commented on content/en/blog/2023/jaeger-exporter-collector-migration.md:
```diff
@@ -0,0 +1,73 @@
+If either of these options do not work for your use-case, please reach out to
+the #otel-collector community via [CNCF Slack](https://slack.cncf.io) or open an
+issue in the
+[repository](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues).
```
```suggestion
[repository](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/new/choose).
```

svrnm commented on content/en/blog/2023/jaeger-exporter-collector-migration.md:
```diff
@@ -0,0 +1,73 @@
+---
+title: Migrating away from the Jaeger exporter in the Collector
+linkTitle: Migrating away from the Jaeger exporter in the Collector
```
```suggestion
linkTitle: Jaeger Collector Exporter Migration
```

The `linkTitle` needs to be fairly short, not 100% happy with my suggestion, but it's the first that came to my mind

svrnm reviewed (changes requested):
I am happy to have it as a blog post to announce and promote it better, but eventually this should be placed somewhere here: https://opentelemetry.io/docs/migration/#jaeger-client

codeboten commented on content/en/blog/2023/jaeger-exporter-collector-migration.md:
```diff
@@ -0,0 +1,73 @@
+linkTitle: Jaeger Collector Exporter Migration
+date: 2023-09-14
+author: >-
+  [Alex Boten](https://twitter.com/codeboten) (Lightstep)
```
```suggestion
  [Alex Boten](https://github.com/codeboten) (Lightstep)
```

codeboten commented:
> I am happy to have it as a blog post to announce and promote it better, but eventually this should be placed somewhere here: https://opentelemetry.io/docs/migration/#jaeger-client

Right the documentation already covers how to use OTLP to send data to Jaeger since https://github.com/open-telemetry/opentelemetry.io/pull/3262, I'm happy to move this content wherever it makes the most sense in the future.

yurishkuro commented on content/en/blog/2023/jaeger-exporter-collector-migration.md:
````diff
@@ -0,0 +1,73 @@
+```yaml
+exporters:
+  otlp/jaeger: # Jaeger supports OTLP directly. The default port for OTLP/gRPC is 4317
+    endpoint: https://jaeger.example.com:4317
````
TLS is not enabled in Jaeger by default, it's worth calling out here.

codeboten commented on content/en/blog/2023/jaeger-exporter-collector-migration.md:
````diff
@@ -0,0 +1,73 @@
+    exporters: [otlp/jaeger]
+```
+
+Depending on your version of Jaeger, you may need to set the following
+environment variable `COLLECTOR_OTLP_ENABLED=true`.
````
```suggestion
Versions of Jaeger until v1.46.0, need the following environment
variable set `COLLECTOR_OTLP_ENABLED=true`.
```

cartermp commented:
@codeboten if you run `npm run test-and-fix` then this should go green.

codeboten commented:
@cartermp what should i do with the refcache check?

cartermp commented:
Hmm, did `npm run test-and-fix` not work? That should be the fix.

codeboten commented:
so test-and-fix didn't show any changed files.... and running fix:format gives me a bunch of unrelated changes:

```
        modified:   assets/js/registrySearch.js
        modified:   content/en/blog/2022/instrument-apache-httpd-server/index.md
        modified:   content/en/docs/instrumentation/js/getting-started/browser.md
        modified:   content/en/docs/instrumentation/js/manual.md
        modified:   content/en/docs/instrumentation/js/propagation.md
        modified:   content/en/docs/instrumentation/js/serverless.md
        modified:   gulp-src/_md-rules/trim-code-block-and-unindent.js
        modified:   gulp-src/prune.js
        modified:   scripts/registry-scanner/index.mjs
```

Do i have to commit thouse?

codeboten commented:
Thanks @cartermp!
